config/openiotsdk/cmake/sdk.cmake

```diff
@@ -50,7 +52,7 @@ FetchContent_Declare(
 FetchContent_Declare(
     trusted-firmware-m
     GIT_REPOSITORY  https://git.trustedfirmware.org/TF-M/trusted-firmware-m.git
-    GIT_TAG         d0c0a67f1b412e89d09b0987091c12998c4e4660
+    GIT_TAG         TF-Mv1.8.0
```

Comment:
Do we still need the `FetchContent_Declare` override ?

Reply:
We need to apply the patch for GCC 12 compiler.

Reply:
Thanks, I had forgotten there is no variable to pass to the SDK to patch TFM